main.cpp

```diff
@@ -475,26 +587,39 @@ Thing 9) fuel system
  */
 struct FuelSystem
 {
-    //injection pressure
+    
     int injectionPressure = 60;
-    //type
     bool typeIsDiesel = true;
-    //capacity
     double capacity = 34.3;
-    //filter amount
     float filterAmount = 33.5f;
-    //pump power
     int pumpPower = 23;
 
-    //power engine
-    float powerEngine(double CC);
-    //spark plugs
+    
+    void powerEngine(double CC);
     void sparkPlugs(int number);
-    //not freeze at 0 degrees
     void freezeTemp(int whatDegree);
 
 };
 
+ void FuelSystem::powerEngine(double CC)
+{
+    std::cout << CC << std::endl;
+}
+
+void FuelSystem::sparkPlugs(int number)
+{
+    number = 10;
+}
```

Comment:
Fix this according to my previous pull request review instructions
- [ ] revise